src/Features/Core/Portable/ExtractInterface/AbstractExtractInterfaceService.cs

```diff
@@ -393,7 +393,8 @@ private static ImmutableArray<ISymbol> CreateInterfaceMembers(IEnumerable<ISymbo
                             modifiers: new DeclarationModifiers(isAbstract: true),
                             type: @event.Type,
                             explicitInterfaceImplementations: default,
-                            name: @event.Name));
+                            name: @event.Name, 
+                            documentationCommentXml: @event.GetDocumentationCommentXml()));
```

Comment:
@CyrusNajmabadi Maybe you're a good person to ask about this? The comment xml gets everything including the id string. Is there a simple way to get just the summary text? I don't see anything in the existing extension methods or in the DocumentationComment object. 

Am I missing anything?